walnut/converter/converter.go

```diff
@@ -175,21 +189,57 @@ func GenerateTaskTemplate(extractedTasks []*task.Class, dumpFileName string) (er
 	return
 }
 
-func GenerateWorkflowTemplate(input workflow.Role, dumpFileName string) (err error) {
-	dir, _ := os.Getwd()
-	path := filepath.Join(dir, dumpFileName, "workflows")
+// confirmOverwrite takes an input string message to show the user and receive a yes/no response
+// if anything other than y/n provided, keep asking
+func confirmOverwrite (ask string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Printf("%s [y/n]", ask)
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+		response = strings.ToLower(strings.TrimSpace(response))
+
+		if response == "y" || response == "yes" {
+			return true
+		} else if response == "n" || response == "no" {
+			return false
+		} else {
+			fmt.Printf("Invalid response\n")
+		}
+	}
```

Comment:
Your logic is correct but I think we can & should do better than this manual implementation.

Please take a look at https://github.com/AlecAivazis/survey which popped up after a quick search. It handles automatically yes-no questions, as well as many other kinds of interactive terminal input.

You can also look for an alternative package in https://awesome-go.com/ but don't spend too much time on this if `AlecAivazis/survey` turns out to work well.

Reply:
I chose to go with: github.com/manifoldco/promptui